meerk40t/lihuiyu/gui/lhycontrollergui.py

```diff
@@ -356,7 +367,7 @@ def on_connection_state_change(self, origin, state):
             self.button_device_connect.SetLabel(_("Connect failed"))
             self.button_device_connect.SetBitmap(
                 icons8_disconnected.GetBitmap(
-                    use_theme=False, resize=get_default_icon_size()
+                    use_theme=False, resize=0.75 * get_default_icon_size()
```

Comment:
You probably want to set `factor = 0.75` in there and just use that since otherwise these numbers are more fully hardcoded. Not that I'm recommending hardcoding them, but making it all adjustable in code from one line.

Reply:
Done